README.md

```diff
@@ -24,52 +22,72 @@ To get a development environment running on your local machine:
 cp .env.example .env
 ```
 
-3. Install packages
+### Install packages
 
 ```bash
 npm install
 ```
 
-4. Database Setup
-   You'll need to create a free database with [Xata](https://xata.io/) which runs on Postgres. When you create your db, make sure enable the checkbox that says `Enable direct access to Postgres`.
+### Database Setup
+
+To get started, you'll need to create a **free** account with [Xata](https://xata.io/).
+
+Xata runs on Postgres, but to get access to the Postgres layer, youll need to enable this in your workspace settings. Find your workspace settings and toggle to enable `Direct Access to Postgres (BETA)`.
+
+Now, you'll need to create a new database. Give it a name, and make sure to enable the checkbox that says `Enable direct access to Postgres`.
+
+> In this project, we'll be connecting to our db in two different ways: using Prisma ORM and the Xata client SDK. All standard CRUD (Create, read, update, and delete) db interactions will be run through Prisma. We'll only be using the Xata client to take advantage of Xata specific features like search and file uploads.
+
+Then, select your newly created database. On the next page, you'll want to copy the `PostgreSQL endpoint` property. It will look something like this.
+
+`postgresql://l5kbra:<YOUR_API_KEY>@us-east-1.sql.xata.sh/<DB_NAME>:main?sslmode=require`
 
-> Since we'll be using Prisma to interact with our database, you need to `Enable direct access to Postgres`.
+> Xata has branches to create multiple instance of a database. The default branch is `main`. These onboarding steps will assume you are using this default `main` branch. Learn more about [Xata branching](https://xata.io/branching).
 
-Then, select your newly created database. On the next page, you'll want to copy the `PostgreSQL endpoint` property. Then add this property to your `.env` file like so.
+You'll need to replace `<YOUR_API_KEY>` with a new API key. You can create your API key in your `Account Settings` which can be found in the dropdown menu on your profile icon in the top right corner of the dashboard.
 
-> You'll need to also create an API Key to include in this DB URL. You can do this in your `account settings`.
+Then, you'll need to update the `DATABASE_URL` environment variable in your `.env` file.
 
 ```bash
-DATABASE_URL=<your_postgresql endpoint>
+DATABASE_URL=postgresql://l5kbra:<YOUR_API_KEY>@us-east-1.sql.xata.sh/<DB_NAME>:main?sslmode=require
```

Comment:
Move this higher up, below the `DATABASE_URL` note

Reply:
Not clear on this one as well. What `DATABASE_URL` note are you referring to?

Reply:
The one on line 43